syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -281,6 +281,10 @@ def mul_and_div(self, other, cmd):
         which is inherent to these operations in the fixed precision setting
         """
         changed_sign = False
+        if isinstance(other, float):
+            other = torch.tensor(other).fix_prec(precision_fractional=self.precision_fractional)
```

Comment:
Maybe this is better to have all arguments of `self`
```suggestion
            other = torch.tensor(other).fix_prec(**self.get_class_attributes())
```